pkg/gen/input/workflows/internal/file/run_ossf_scorecard.yaml.template

```diff
@@ -0,0 +1,75 @@
+{{{{ .Header }}}}
+
+# This workflow uses actions that are not certified by GitHub. They are provided
+# by a third-party and are governed by separate terms of service, privacy
+# policy, and support documentation.
+
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '15 15 15 * *'
+  push:
+    branches: [ "main" ]
```

Comment:
Unfortunately, quite some repos still have their default branch named `master`

Reply:
Ah yes I didn't think about that. It wouldn't be great to have to specify the branch manually for each repo either.
When I return I'll see about detecting the default branch

Reply:
I checked quickly and it is possible ot detect the default branch through the GitHub API (`gh repo view --json defaultBranchName`) but it also works to simply specify both branches, as long as there are only pushes to the actual default branch. In other words, the action will complain if the default branch is `main` but there are additional pushes to `master`. I will update this to list both branches